.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,32 @@
-## FreeCodeCamp Pull Request template
-Please, go through these steps before you submit a PR.
+<!-- FreeCodeCamp Pull Request Template -->
+
+<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
+<!-- Make sure that your PR is not a duplicate -->
+
+#### Pre-Submission Checklist
+<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
+<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
```

Comment:
@raisedadead I'm not sure why it doesn't show up above but it is added if you look at the file 😄